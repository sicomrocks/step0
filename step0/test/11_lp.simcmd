#Test de la fonction lp : fonctionnement normal

lp ./res/exempleELF.o
