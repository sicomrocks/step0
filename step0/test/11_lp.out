
------ Fichier ELF "./res/exempleELF.o" : sections lues lors du chargement ------
